src/Trigger/RabbitMQTriggerAttribute.cs

```diff
@@ -9,28 +9,31 @@ namespace Microsoft.Azure.WebJobs
     [Binding]
     public sealed class RabbitMQTriggerAttribute : Attribute
     {
-        public RabbitMQTriggerAttribute(string connectionStringSetting, string queueName, string deadLetterExchangeName = "")
+        public RabbitMQTriggerAttribute(string connectionStringSetting, string queueName, string deadLetterExchangeName = "", ushort batchNumber = 1)
         {
             ConnectionStringSetting = connectionStringSetting;
             QueueName = queueName;
             DeadLetterExchangeName = deadLetterExchangeName;
+            BatchNumber = batchNumber;

```

Comment:
I don't understand BatchNumber - should this even be part of the attribute? Given that you were flowing it incorrectly as a prefetch value